pyradiosky/skymodel.py

```diff
@@ -2321,7 +2489,13 @@ def read_skyh5(
                     if parname not in header:
                         continue
 
-                dset = header[parname]
+                if parname == "lat":
+                    dset = header["dec"]
+                elif parname == "lon":
+                    dset = header["ra"]
+                else:
+                    dset = header[parname]
+
```

Comment:
we should be looking for the generic lont/lat instead of ra/dec here. We'll need to have special handling for file names with ra/dec since those will be older file types.